adcc/adc_pp/state2state_transition_dm.py

```diff
@@ -20,12 +20,131 @@
 ## along with adcc. If not, see <http://www.gnu.org/licenses/>.
 ##
 ## ---------------------------------------------------------------------
+from adcc import block as b
 from adcc.AdcMethod import AdcMethod
+from adcc.functions import einsum
 from adcc.AmplitudeVector import AmplitudeVector
+from adcc.OneParticleOperator import OneParticleOperator
 
 import libadcc
 
-DISPATCH = {}  # None implemented
+
+# TODO: just a sketch for what I'd like to have, because we need to check on
+# 2 amplitude vectors simultaneously
```

Comment:
These functions should not be here. I think they are useful, though. I suppose they should go to some sort of util or common file in adc_pp.

Reply:
Also can you redirect the checks from the other implementations (MTM, other densities) to these new functions.

Reply:
Sure. MTMs don't involve any AmplitudeVectors though 😉

Reply:
Done.